packages/grafana-ui/src/components/Icon/Icon.tsx

```diff
@@ -1,11 +1,18 @@
-import React from 'react';
-import { cx, css } from 'emotion';
-import { stylesFactory } from '../../themes';
+import React, { PureComponent, ComponentType } from 'react';
+import { css, cx } from 'emotion';
+
+import { stylesFactory, withTheme } from '../../themes';
+import { Themeable } from '../../types';
 import { IconType } from './types';
 
-export interface IconProps {
+type Size = 'xs' | 'sm' | 'base' | 'md' | 'lg';
```

Comment:
not sure about the 'base'. think we should use same type as button size: https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/Button/types.ts#L5 

Think we should move to grafana-ui/types and name it ComponentSize so we can share that type between components

Reply:
Yeah, that is a good idea! I'll do that then.